components/automate-gateway/handler/infra_proxy/servers.go

```diff
@@ -37,6 +38,26 @@ func (a *InfraProxyServer) GetServer(ctx context.Context, r *gwreq.GetServer) (*
 	}, nil
 }
 
+// ValidateWebuiKey validates the webui key
+func (a *InfraProxyServer) ValidateWebuiKey(ctx context.Context, r *gwreq.ValidateWebuiKey) (*gwres.ValidateWebuiKey, error) {
+	req := &infra_req.ValidateWebuiKey{
+		Id:         r.Id,
+		Fqdn:       r.Fqdn,
+		WebuiKey:   r.WebuiKey,
+		IsWebuiKey: r.IsWebuiKey,
```

Comment:
We should pass it as true and no need to get it from request:
```suggestion
		IsWebuiKey: true,
```